pkgs/applications/science/misc/bada-bib/default.nix

```diff
@@ -9,23 +9,23 @@
 , glib
 , gnome
 , gobject-introspection
-, gtk3
+, gtk4
 , libxml2
 , pkg-config
 , python3Packages
 , wrapGAppsHook }:
```

Comment:
We really want to use `wrapGAppsHook4` for all GTK4 apps :crying_cat_face:
Submitted https://github.com/RogerCrocker/BadaBib/pull/3

Reply:
Ah thanks, missed that :)

Reply:
The patch has been accepted, do you mind fetchpatch the patch and use `wrapGAppsHook4`?

Reply:
Hi, I thought this issues might affect others as well, so I just made a new release with your patch included.

Reply:
Thanks :smile: